.travis.yml

```diff
@@ -1,24 +0,0 @@
-language: node_js
-install:
-  - npm install -g codecov
```

Comment:
@ephread why did we need the global install in travis only again? Should I add this line in the github actions, or could we specify it as a dev dep?

Reply:
I wouldn't put it in the dev dependencies, because we don't really need Codecov's uploader locally (we're never going to upload coverage). I'm not sure it needs to be installed globally though, I guess it was easier at the time!